tests/publiccloud/boottime.pm

```diff
@@ -423,7 +423,7 @@ sub run {
 
     my $results = $self->measure_timings($args);
     $self->store_in_db($results) if (check_var('_PUBLIC_CLOUD_PERF_PUSH_DATA', 1));
-    $self->check_thresholds($results);
+    $self->check_thresholds($results) if (check_var('PUBLIC_CLOUD_PERF_THRESH_CHECK'));
```

Comment:
please change it back it won't work , you need to pass 1 here

Reply:
Sorry, I have read late your reply, after I changed it.

Reply:
But I could use directly **get_var**, any *not null* value is ok:
` ... if (get_var('PUBLIC_CLOUD_PERF_THRESH_CHECK'))
`